.github/workflows/tests.yaml

```diff
@@ -16,6 +16,13 @@ jobs:
         uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python-version }}
+      - name: Install Docker Compose v2
```

Comment:
I'd like to give this action a try https://github.com/ndeloof/install-compose-action

I'll reach you back with my findings